hack/images/image.list.load.online.example

```diff
@@ -26,4 +26,4 @@ karmada-scheduler-estimator;docker.io/karmada/karmada-scheduler-estimator:v1.9.0
 karmada-aggregated-apiserver;docker.io/karmada/karmada-aggregated-apiserver:v1.9.0;;;
 karmada-search;docker.io/karmada/karmada-search:v1.9.0;;;
 karmada-metrics-adapter;docker.io/karmada/karmada-metrics-adapter:v1.9.0;;;
-karmada-agent;docker.io/karmada/karmada-agent:v1.9.0;;;
+karmada-agent;docker.io/karmada/karmada-agent:v1.9.0;;;
```

Comment:
No end of the line. not supposed to change.
But no big deal :)

Reply:
I am trying to squash commits.🙂